Presentation.Web/Models/API/V1/UI_Configuration/CustomizedUINodeDTO.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Configuration;
+using System.Linq;
+using System.Web;
+
+namespace Presentation.Web.Models.API.V1.UI_Configuration
+{
+    public class CustomizedUINodeDTO
+    {
+        [Required]
+        [RegexStringValidator("^([a-zA-Z]+)(\\.[a-zA-Z]+)*$")]
+        public string FullKey { get; set; }
```

Comment:
perhaps just call it key here.. in the UI FullKey makes sense but here it is just a key... my bad. I will change it in ui